src/main/java/seedu/address/model/export/DocumentPath.java

```diff
@@ -100,11 +105,23 @@ public String toString() {
         return directoryPath.toString() + File.separator + documentFilePath.toString();
     }
 
+    @Override
+    public void export(List<FlashCard> list) throws IOException {
+        try {
+            DocumentExportUtil.exportFlashCardsToDocument(
+                    list, this
+            );
+        } catch (IOException e) {
+            throw e;
+        }
+    }
+
     /**
      * Converts this DocumentPath into a String representing its absolute path
      *
      * @return String representing the absolute path of this DocumentPath
      */
+    @Override
     public String toAbsolutePathString() {
         return this
                 .getPath()
```

Comment:
Similarly you can just `return getPath();`

Reply:
I'll keep this one so that all the method calls remain on the same indentation level.